### ISSUE TITLE
Can't start puppet-forge-server

### ISSUE THREAD

**Issue by danjng** [unibet/puppet-forge_server#2]
After fixing my previous issue, I was able to get further in terms of Puppet run status, but still encounter the following:
`Error: Could not start Service[puppet-forge-server]: Execution of '/sbin/service puppet-forge-server start' returned 1: Starting puppet-forge-server: bash: puppet-forge-server: command not found
[FAILED]
Error: /Stage[main]/Forge_server::Service/Service[puppet-forge-server]/ensure: change from stopped to running failed: Could not start Service[puppet-forge-server]: Execution of '/sbin/service puppet-forge-server start' returned 1: Starting puppet-forge-server: bash: puppet-forge-server: command not found
[FAILED]`

If I try to run it manually, I get the same result:
`Starting puppet-forge-server: bash: puppet-forge-server: command not found
                                                           [FAILED]`

Any ideas? I believe dependencies are good, and I have create symbolic links for `ruby` and `gem` from `/opt/puppet/bin`.

**Comment by i11** [comment]
It seems like you're missing gem executable directory in `PATH` variable. This module handles `PATH` variable in case of SCL setup, otherwise it's up to you to properly prepare ruby environment. You could run `gem environment` and checkout your `EXECUTABLE DIRECTORY`. Most probably _puppet-forge-server_ binary will be there in that case make sure it's included in `PATH` variable.

**Comment by danjng** [comment]
Thanks again, i11!  You're 2 for 2! I appreciate the help!

I've actually forked and made some basic modifications to cover a scenario in which pe_gem can be set as a provider and the appropriate path (/opt/puppet/bin) is set as the path.  I've also made a change to allow proxying to be disabled (right now, whether you specify something or not, it will always proxy to Puppetlabs' instance).  Will create a pull request for your review.
